FlowCrypt/Controllers/Setup/SetupCreatePassphraseViewController.swift

```diff
@@ -263,13 +312,17 @@ extension SetupGenerateKeyViewController {
             return
         }
         logger.logInfo("Setup account with \(passPhrase)")
-        setupAccountWithGeneratedKey(with: passPhrase)
+        if keys.isNotEmpty {
+            self.setupAccountWithExistingKeys(with: passPhrase)
+        } else {
+            setupAccountWithGeneratedKey(with: passPhrase)
+        }
```

Comment:
This decision will happen in the VC that fetched the keys from EKM - in EKM flow it will use the dedicated EKM VC.